### ISSUE TITLE
Hide track name for unnamed tracks

### ISSUE THREAD

**Issue by Banjobeni** [Cappycot/DVDispatcherMod#5]
I would be nice to hide track names like "#S704#T". They don't give more info to the player if being shown.

On a larger scale, for "current job cars" or later maybe "current step cars", multiple cars can be on different tracks, even in the same consist. Cars on tracks having meaningful names should be priorized for giving track name, so that we only hide track information if we really have nothing better to display.

previously discussed in #4

**Comment by Banjobeni** [comment]
I started working on this. The actual filtering of unnamed tracks is easy, but I took the opportinuty to change the attention point to the center car of the "adjacent car group" instead of the center car of the whole consist. This helps identify cars if multiple jobs have cars in the same consist or if the game wants you to fragment an emptied consist into tiny pieces.

Thereby I accidentially dropped the feature that the "last player loco" is the attention point in some cases. As a non-VR player I don't get much info from that, but I could imagine this being different in VR? What are your stakes on that?
